docs/src/main/tut/user-guide.md

```diff
@@ -441,7 +441,7 @@ For all `String`-based endpoints, Finch provides an `as[A]` method to perform ty
 is available for any `Endpoint[String]`, `Endpoint[Option[String]]` or `Endpoint[Seq[String]]` as
 long as a matching implicit `io.finch.DecodeEntity[A]` type-class is in the scope.
 
-This facility is designed to be intuitive, meaning that you do not have to provide a
+This facility is designed to be intuitive, meaning that you do not have to provide an
```

Comment:
Okay, I reverted these changes, and left `io.finch` as is for consistency. I'd like to let others to do the dropping `io.finch` parts (since I'm not so sure which one to drop and which one to keep).